### PR TITLE
Add recipe for cardiac-mps

### PR DESCRIPTION
**Pull request by finsberg**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by finsberg** [comment]
@conda-forge/python ready for review!

File: recipes/cardiac-mps/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  commands:
+    - pip check
+    - mps --help
+
+about:
+  home: https://github.com/ComputationalPhysiology/mps
+  summary: Tools for working with cardiac mps files
+  license: LGPL-2.1-or-later
+  license_family: LGPL
+  license_file: LICENSE
```
Comment (dopplershift) [inline recipes/cardiac-mps/meta.yaml:57]: Need to include the licenses that are in the `licenses/` directory. You can use an expression (appropriate for all 3) like: `LGPL-2.1-or-later AND BSD-3-Clause AND BSD-2-Clause`

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just clean up the license stuff and this is good to go.

File: recipes/cardiac-mps/meta.yaml (hunk 1 above)
Comment (finsberg) [reply recipes/cardiac-mps/meta.yaml]: Thanks for pointing that out. This should now be fixed in https://github.com/conda-forge/staged-recipes/pull/22473/commits/b39b66e53760eb6473bcc2151805444645a1fae8

**Review by dopplershift — APPROVED** [review]

**Pull request merged**